src/stan/variational/advi.hpp

```diff
@@ -313,7 +313,7 @@ namespace stan {
 
         if (out_stream_) {
           services::io::write_iteration(*out_stream_, model_, rng_,
-                                        0.0, cont_vector, disc_vector,
+                                        lp, cont_vector, disc_vector,
```

Comment:
@dustinvtran, I think it's about time we found a way to test for output in the variational inference code. We should have a test that verifies this is actually getting out to the `out_stream_`.
How are you currently verifying that this change 1) doesn't break and 2) correctly streams things out? We should just get that into a repeatable test so we don't regress in the future.